README.md

```diff
@@ -27,23 +35,4 @@ Add Docker Compose Ruby SDK in RubyMine settings:
 4. Press `OK`
 5. Select added SDK in `Ruby SDK and Gems` and press `OK`
 
-Next, migrate the database:
```

Comment:
Would be great to keep it just to show how to work with Docker Compose SDK configuration in RubyMine

Reply:
I'll update it to show how to use the Rake tasks.